.github/workflows/cypress-workflow.yml

```diff
@@ -115,17 +115,10 @@ jobs:
           yarn start --no-base-path --no-watch &
         shell: bash
 
-      # Window is slow so wait longer
-      - name: Sleep until OSD server starts - windows
-        if: ${{ matrix.os == 'windows-latest' }}
+      - name: Sleep until OSD server starts
```

Comment:
Why did we remove check for Windows?

Reply:
Windows check still here, just merged different OS types wait in one check